src/shared/codelens/csharpCodeLensProvider.ts

```diff
@@ -463,15 +478,22 @@ async function _installDebugger(
     try {
         const vsdbgPath = getDebuggerPath(targetFolder)
 
+        const samCliContext = getSamCliContext()
+        const samCliVersionValidatorResult = await samCliContext.validator.getVersionValidatorResult()
+        const samCliVersion = samCliVersionValidatorResult.version
```

Comment:
why not put these lines in `getSamCliDockerImageName()` ?

Reply:
Not sure actually, I think I forgot that I added them and that they weren't here to start. Can move.

Reply:
Eh, I'm thinking about it and it tests better this way (we'd have to cut the unit tests with the SAM invocation). If you still have strong inclinations towards combining, I'm happy to do it.